roles/openstack-setup/tasks/flavors.yml

```diff
@@ -21,6 +26,7 @@
       password: "{{ secrets.admin_password  }}"
   with_items: '{{ nova.flavors | default([]) }}'
   run_once: true
+  when: flavorcount.stdout == ""
   # FIXME flavors are always added, even if it's just an Ironic cloud
```

Comment:
if the customer has deleted the flavor intentionally, are we sure that we want to recreate it?